### ISSUE TITLE
Charts should have axis scale info

### ISSUE THREAD

**Issue by kim0** [exelban/stats#70]
Thanks for your recent work adding charts for network info, and improving the visuals everywhere. It looks amazing :) However, it's still hard from the network graph to tell what that spike means, was it 1Mbps or 100Mbps ..etc. I think for graphs like Memory and Network, some kind of numbers on the y-axis are needed. At least what the top value of the graph means. But ideally a couple of more data points (say 33%, 66%, 100%). The CPU might not need it, bec the max is always 100%.

Thanks!

**Comment by exelban** [comment]
Hi. 

I will add it with time. Or maybe someone will help with it. Because it's not critical for me. So it can take some time.

**Comment by kim0** [comment]
Absolutely, thanks a lot for the time you invest in this app. Really appreciating your time and effort. Cheers mate

**Comment by exelban** [comment]
Hi. 

Ok, I don't like this chart at all.
So I redraw it. Also, I add the information about the maximum visible value in the chart.

![screenshot](https://user-images.githubusercontent.com/13332412/90554445-29bfc880-e196-11ea-841c-829d313b9b38.png)

It will be available with the next release.

**Comment by kim0** [comment]
Great stuff! Your work is awesome. Thanks a lot.

If I may suggest adding a light horizontal line at the 50% mark, would make
this much easier to read. Although it's already a huge improvement! 👍
